httpie/input.py

```diff
@@ -630,12 +646,15 @@ def parse_items(items,
         elif item.sep == SEP_QUERY:
             target = params
         elif item.sep == SEP_FILES:
-            try:
-                with open(os.path.expanduser(value), 'rb') as f:
-                    value = (os.path.basename(value),
-                             BytesIO(f.read()))
-            except IOError as e:
-                raise ParseError('"%s": %s' % (item.orig, e))
+            if(value == '-'):
```

Comment:
Please remove these parentheses. It is not the style of the project and it is not typical to see in Python.